test/functional/ui/inccommand_user_spec.lua

```diff
@@ -253,6 +253,61 @@ describe("'inccommand' for user commands", function()
     ]]
   end)
 
+  it("restores 'modifiable'", function()
```

Comment:
Don't need separate tests. The "can preview nomodifiable" test can also edit the buffer and assert the stuff this test is doing.